recipes/lasio/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "lasio" %}
+{% set version = "0.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kinverarity1/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 140eab1b74b63374bc2d1a96f83143823d3432f84bb32311071c40bb22f59d9c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ PKG_VERSION }}
+  number: 0
+  entry_points:
+    - las2excel = lasio.excel:main
+    - las2excelbulk = lasio.excel:main_bulk
+    - lasversionconvert = lasio.convert_version:convert_version
+    - lasio = lasio:version
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - setuptools_scm
+    - pip
+  run:
+    - python >=3.7
+    - pandas
+    - chardet
+    - openpyxl
+
+test:
+  imports:
+    - lasio
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/kinverarity1/lasio
+  summary: 'Read/write well data from Log ASCII Standard (LAS) files'
+  description: |
+    This is a Python 3.7+ package to read and write Log ASCII Standard (LAS) files, used for borehole data such as geophysical, geological, or petrophysical logs.
```

have a white line to force a line break
```suggestion
    This is a Python 3.7+ package to read and write Log ASCII Standard (LAS) files, used for borehole data such as geophysical, geological, or petrophysical logs.

```